playbooks/engineer-workflow.md

```diff
@@ -142,6 +143,15 @@ Further reading:
 - [Rules of communicating at GitHub](https://ben.balter.com/2014/11/06/rules-of-communicating-at-github/)
 - [RFC on defining PR roles at Artsy](https://github.com/artsy/README/issues/177)
 
+### Code Reviews
+
+The best PR comments are clear in their intention. While we don't mandate that PR comments follow any specific
+format, some people have found it helpful to adopt the standards outlined in
+[Conventional Comments](https://conventionalcomments.org).
+
+In particular, we encourage the use of `blocking` and `non-blocking` to signal whether or not a question or comment
+should be resolved before merging.
```

Comment:
👍  This could work well with Github's "Request changes" feature if we want to make blocking comments more obvious.